_shared/hooks/post_gen_project.py

```diff
@@ -26,6 +26,13 @@ def remove_conditional_files():
         "conf/alembic.ini",
         "{{ cookiecutter.package_name }}/migrations/env.py",
         "{{ cookiecutter.package_name }}/migrations/script.py.mako",
+        "{{ cookiecutter.package_name }}/db.py",
+        "{{ cookiecutter.package_name }}/models/__init__.py",
+        "tests/factories/__init__.py",
+        "tests/factories/factoryboy_sqlalchemy_session.py",
+        "tests/pytest_plugins/db.py",
+        "tests/unit/pytest_plugins/db.py",
+        "tests/functional/pytest_plugins/db.py",
```

Comment:
Adding these files to this list means that they won't be added to projects that don't use the `postgres` option.